io/js/src/test/scala/fs2/io/NodeJSCompressionSuite.scala

```diff
@@ -22,10 +22,17 @@
 package fs2
 package io
 
-import fs2.CompressionSuite
+import cats.effect.IO
+import fs2.compression.{Compression, DeflateParams, InflateParams, ZLibParams}
 import fs2.internal.jsdeps.node.zlibMod
 import fs2.io.compression._
 import fs2.io.internal.ByteChunkOps._
+import org.scalacheck.effect.PropF.forAllF
+import scodec.bits.{ByteVector, crc}
+
+import java.nio.charset.StandardCharsets
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.FiniteDuration
 
 class NodeJSCompressionSuite extends CompressionSuite {
```

Comment:
Oh, also if you've added any additional tests here would be cool move them into `CompressionSuite` to run them on JVM as well :)

Reply:
Tests are now shared :) (including that couple of new `unix gzip -> gunzip`)